internal/testutil/pgpkeys.go

```diff
@@ -0,0 +1,182 @@
+package testutil
+
+import (
+	"log"
+
+	"golang.org/x/crypto/openpgp/packet"
+
+	"github.com/canonical/chisel/internal/openpgputil"
+)
+
+type PGPKeyData struct {
+	ID           string
+	PubKeyArmor  string
+	PrivKeyArmor string
+	PubKey       *packet.PublicKey
+	PrivKey      *packet.PrivateKey
+}
+
+var PGPKeys = map[string]*PGPKeyData{
+	"keyUbuntu2018": {
```

Comment:
I believe we typically use string keys as "key-ubuntu-2018" instead of camel case, but please double check.

Reply:
I used mixedCaps because in [Effective Go](https://go.dev/doc/effective_go#mixed-caps), it says:
> MixedCaps
> 
> Finally, the convention in Go is to use MixedCaps or mixedCaps rather than underscores to write multiword names.

Reply:
I have changed it to match the style you proposed which is also used in one more place in the code.

Reply:
Uncommented and the tests are passing.